### PR TITLE
Double sidebar

### PR DESCRIPTION
**Pull request by zeantsoi**
👓 @jamesmaa 

[My other branch](https://github.com/reddit/reddit-plugin-adzerk/pull/242) came detached so this is a continuation.

### CODE REVIEW

**Comment by zeantsoi** [comment]
d28b1ea addresses this comment: https://github.com/reddit/reddit-plugin-adzerk/pull/242#discussion_r78496776

💇

File: reddit_adzerk/adzerkads.py [hunk 1]
```diff
@@ -67,6 +67,9 @@ def __init__(self, frame_id="ad_main", displayed_things=[], link=None):
             if percentage is not None:
                 data["properties"]["percentage"] = percentage
         self.ad_url = g.adzerk_url.format(data=json.dumps(data))
+        
+        print 'ad_url: %s' % self.ad_url
+        print "FUCK FUCK FUCK"
```
Comment (jamesmaa) [inline reddit_adzerk/adzerkads.py]: 😳

Comment (zeantsoi) [inline reddit_adzerk/adzerkads.py]: :satisfied: glad we have code reviews.

Hey I'm gonna close this since I have to bike my branch, I'll reopen later.

**Comment by zeantsoi** [comment]
Closing in order to delete/recreate master repo. I'll reopen this PR there.

**Pull request closed without merging**